source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx

```diff
@@ -233,6 +234,19 @@ export function EditorStateProvider(props: { children: JSXElement }) {
 		return inlang()?.config() ? true : false
 	}
 
+	const [gitOrigin] = createResource(async () => {
+		return parseOrigin({ remotes: await repo.listRemotes() })
+	})
+	createEffect(() => {
+		if (doesInlangConfigExist() === true) {
+			//Non-null assertion operator because gitOrigin/parseOrign always returns a string.
+			//Even if there is no origin, "unknown" is returned as a string.
+			telemetryBrowser.group("repository", gitOrigin()!, {
+				name: gitOrigin(),
+			})
+		}
+	})
+
```

Comment:
@jannesblobel you don't need an effect. 

You can directly perform the group identify at the point where the repository is opened, in the createResource which calls `openRepository`. 

Your code likely also contains a bug. calling repo without a getter `repo()` is not reactive